dockstore-integration-testing/src/test/java/io/dockstore/webservice/NotebookIT.java

```diff
@@ -214,7 +215,30 @@ void testStarringNotebook() {
         notebook = workflowsApi.getWorkflow(notebookID, "");
         assertEquals(0, notebook.getStarredUsers().size());
     }
+    @Test
+    void testNotebookRSSFeedAndSitemap() {
+        CommonTestUtilities.cleanStatePrivate2(SUPPORT, false, testingPostgres);
+        ApiClient openApiClient = getOpenAPIWebClient(BasicIT.USER_2_USERNAME, testingPostgres);
+
+        // There should be no notebooks
+        assertEquals(0, notebookDAO.findAllPublishedPaths().size());
+        assertEquals(0, notebookDAO.findAllPublishedPathsOrderByDbupdatedate().size());
+
+        CreateContent createContent = new CreateContent().invoke();
+        long notebookID = createContent.getNotebookID();
+
+        // There should be 1 notebook
+        assertEquals(1, notebookDAO.findAllPublishedPaths().size());
+        assertEquals(1, notebookDAO.findAllPublishedPathsOrderByDbupdatedate().size());
+
+        final MetadataApi metadataApi = new MetadataApi(openApiClient);
+        String rssFeed = metadataApi.rssFeed();
+        System.out.println(rssFeed);
```

Comment:
Looks like a debugging statement? It doesn't hurt, but not really adding anything.